modules/ROOT/pages/afm-modify-schedule.adoc

```diff
@@ -5,16 +5,14 @@ endif::[]
 
 :imagesdir: ../assets/images
 
-You can modify a schedule if you want to change:
+You can modify a schedule to change:
 
-* The test version that it runs
-* The location in which the test is run from
+* The location at which the test is run
 * The frequency at which to run the test
-* The monitor in which the test is defined
 
 == Before You Begin
 
-If you want to modify a schedule to use a public location instead of a private one, ensure that your organization in Anypoint Platform has not reached its maximum number of schedules for public locations. An indicator at the top of the *Monitors* section gives the count of such schedules that are already being used.
+To run a schedule in a public location, ensure that your Anypoint Platform organization has not reached its maximum number of schedules. An indicator is displayed when creating a new monitor if there are no public schedules available.
```

Comment:
```suggestion
To run a schedule in a public location, ensure that your Anypoint Platform organization has not reached its maximum number of schedules. When you attempt to create a new monitor, an indicator displays if no public schedules are available.
```

Reply:
What do you mean by "an indicator"? An error? A message?

Reply:
Good call out. I replaced "indicator" with "message".